libkbfs/reporter_kbpki.go

```diff
@@ -302,6 +299,8 @@ func (r *ReporterKBPKI) send(ctx context.Context) {
 				stagedStatus = nil
 			}
 			suppressed = false
+		case <-ctx.Done():
```

Comment:
You can now remove all the `if !ok {  return }` lines above, unless you put the above channel `close`s back in.

Reply:
ah now I'm realizing the "before" is better, as channel selection is random.